src/program.cpp

```diff
@@ -145,6 +145,31 @@ String RemoveCommas(String line)
     return returnString;
 }
 
+TokenList RemoveComment(TokenList tokens)
+{
+    TokenList tokensNoComments; 
+    for (size_t i = 0; i < tokens.size(); i++) {
+        if (tokens.at(i)->Content != "#") {
+            tokensNoComments.push_back(tokens.at(i));
+        } else {
```

Comment:
nit: the else should also be placed on its own line

Reply:
fixed